app/policies/organisation_policy.rb

```diff
@@ -30,11 +30,7 @@ def download?
 
   class Scope < Scope
     def resolve
-      if user.administrator?
-        scope.all
-      else
-        scope.where(id: user.organisation_id)
-      end
+      scope.all
```

Comment:
Given that this class isn't doing any significant filtering, is this still necessary to keep pundit magic functioning?